gui/src/main/java/org/useware/kernel/gui/behaviour/FilteringStatementContext.java

```diff
@@ -50,21 +51,9 @@ public String resolve(String key) {
     @Override
     public LinkedList<String> collect(String key) {
         LinkedList<String> actualCollection = delegate.collect(key);
-        String filtered = filter.filter(key);
```

Comment:
I think this is a followup to SecurityFrameworkImpl.java#103 where values are swapped with tuples. As such it should not be necessary